src/polytopes/chain.jl

```diff
@@ -33,6 +33,17 @@ function vertices(c::Chain)
   end
 end
 
+"""
+    segments(chain)
+
+Return the segments linking consecutive points of the `chain`.
+"""
+function segments(c::Chain)
+  vs = vertices(c)
```

Comment:
I think in this case we need to use `c.vertices` to always get all the points including the last one which may be equal to the first one in case of closed chains. The function `vertices(c)` does a more complicated logic to remove the repeated one.